packages/design-system/src/components/Skeleton/variations/SkeletonInput.module.scss

```diff
@@ -0,0 +1,11 @@
+@use '~@talend/design-tokens/lib/tokens';
+@use '../../Form/Primitives/mixins' as Form;
+
+.skeletonInput {
```

Comment:
I think for BEM selectors we usually write them with hyphenated names, not camel case ones
```suggestion
.skeleton-input {
```

Reply:
indeed, I c/c some code from @Benrajalu